presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -762,8 +804,9 @@ public void onFailure(Throwable throwable)
 
     private void processConnectorCommitHandle(Object result)
     {
-        if (result instanceof ConnectorCommitHandle) {
-            addSerializedCommitOutputToOutput((ConnectorCommitHandle) result);
+        if (result instanceof List) {
```

Comment:
why this is changed from `ConnectorCommitHandle` to `List`? Do we have unit test to cover this branch?

Reply:
I made a change and forget to fix this logic. Will do.